src/NHSD.BuyingCatalogue.Organisations.Api/Repositories/OrganisationRepository.cs

```diff
@@ -27,6 +27,20 @@ public async Task<Organisation> GetByIdAsync(Guid id)
             return await _context.Organisations.FirstOrDefaultAsync(org => org.OrganisationId == id);
         }
 
+        public async Task<Organisation> GetByOdsCodeAsync(string odsCode)
+        {
+            return await _context.Organisations.FirstOrDefaultAsync(org => org.OdsCode == odsCode);
```

Comment:
Case will only matter in the SQL if the database is configured with a case-sensitive collation (it's not at the moment as we've no requirement for it).

However, I guess whether we tweak this at all depends on if we want to 'belt-and-brace' it anyway...

Reply:
The casing in the SQL would only matter if we configured the database using a case-sensitive collation. As we're not, we'd only need to tweak this if we want to belt-and-brace it anyway (as it would probably just involve adding a `ToUpperInvariant()`...

Reply:
After some research I came across [this](https://stackoverflow.com/a/28391284/12343768) claiming that the default collation is case insensitive. Being the sceptic I am I went over to SSMS and tried it out. And to my surprise, the stranger on the internet was right. 😄 

Lower:
![image](https://user-images.githubusercontent.com/37105695/79977796-be3f4600-8496-11ea-89fc-05a12d973e79.png)


Upper:
![image](https://user-images.githubusercontent.com/37105695/79977745-a9fb4900-8496-11ea-901d-11228f280f09.png)

To conclude, given our default settings, this should not be an issue.

Reply:
👍